news/2024/2024-11-28-osu-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,135 @@
+
+The match took a left turn with the next two picks. Canada opted into another aim consistency map with [Warcry](https://osu.ppy.sh/beatmapsets/2281532#osu/4864391), but they were stonewalled by a ludicrous display from ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) and ::{ flag=GB }:: [Mahmood](https://osu.ppy.sh/users/7627844), who each FC'd the map, helping the UK pick up a break point, the first of the match. They would immediately give up that advantage on the next pick, as Canada managed to scrape together a 155,339 point win on DT4's awkward high approach rate pick, [Hit & Run](https://osu.ppy.sh/beatmapsets/2281450#osu/4864105).
+
+Tied 4-4, one pick remained per team. Canada selected [My Heart Will Go On](https://osu.ppy.sh/beatmapsets/1865009#osu/3853480) and put on a flow aim clinic, winning it by over 900k. Not to be outdone, the United Kingdom simply set one of the most absurd team scores ever seen in World Cup history — a three-way FC on [Fallen Angel](https://osu.ppy.sh/beatmapsets/36319#osu/403050) from ::{ flag=GB }:: [rudj](https://osu.ppy.sh/users/11592896), ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050), and ::{ flag=GB }:: [Akolibed](https://osu.ppy.sh/users/9269034), all with over 99% accuracy, backed up by a last-second choke from ::{ flag=GB }:: [Mahmood](https://osu.ppy.sh/users/7627844). The final tally? A score of **4,684,571**, one of the highest team scores ever seen on any map in any stage, good for a 1.7 million point win at the most critical moment to force the tiebreaker.
+
+And what a tiebreaker it would be. The map was [today is the day](https://osu.ppy.sh/beatmapsets/2281434#osu/4864081), setting the scene for stalwarts ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) and ::{ flag=CA }:: [xootynator](https://osu.ppy.sh/users/3717598) to go toe-to-toe in a furious battle to advance to the top 6. The two put forth a dazzling display in the match's biggest moment, but ::{ flag=CA }:: [xootynator](https://osu.ppy.sh/users/3717598) found the lone miss at the one-third mark, making up the difference the United Kingdom needed as ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) held over for 2300 combo to secure a win for the UK.
```

Comment:
```suggestion
And what a tiebreaker it would be. The map was [today is the day](https://osu.ppy.sh/beatmapsets/2281434#osu/4864081), setting the scene for stalwarts ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) and ::{ flag=CA }:: [xootynator](https://osu.ppy.sh/users/3717598) to go toe-to-toe in a furious battle to advance to the top 6. The two put forth a dazzling display in the match's biggest moment, but ::{ flag=CA }:: [xootynator](https://osu.ppy.sh/users/3717598) found the lone miss at the one-third mark, making up the difference the ::{ flag=GB }:: United Kingdom needed as ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) held over 2,300 combo to secure a win for the UK.
```